### PR TITLE
:lipstick: [#1334] Align arrows in cards to bottom

### PR DESCRIPTION
**Pull request by jiromaykin**
issue: https://taiga.maykinmedia.nl/project/open-inwoner/task/1334
All arrows need to be 'fixed' and aligned to the bottom of each (product/samenwerken)-Card.
So the arrow-position becomes independent from the description/content height of a card.

### CODE REVIEW

**Comment by jiromaykin** [comment]
@Bartvaderkin @vaszig  After the CMS was implemented a couple of Card styles broke - so in this PR I'm fixing some of those as well (besides the original issue where all arrows need to be fixed to the card bottoms).

**Review by Bartvaderkin — APPROVED** [review]

**Comment by alextreme** [comment]
@vaszig please review this one just in case

File: src/open_inwoner/templates/cms/collaborate/active_plans_plugin.html [hunk 1]
```diff
@@ -5,12 +5,13 @@ <h2 class="h2">
     {% button href="collaborate:plan_list" text=_("Naar samenwerken") icon="arrow_forward" icon_position="after" %}
 </h2>
 {% if plans %}
-<div class="plans-cards card-container card-container--columns-4">
+<div class="plans-cards card-container card-container--columns-4 card-container--user-home">
```
Comment (vaszig) [inline src/open_inwoner/templates/cms/collaborate/active_plans_plugin.html:8]: Do we really need this new class here?I see it follows the same rules as the `card-container--columns-4`

**Review by vaszig — APPROVED** [review]
One small question, besides that it works fine!

File: src/open_inwoner/templates/cms/collaborate/active_plans_plugin.html (hunk 1 above)
Comment (alextreme) [reply src/open_inwoner/templates/cms/collaborate/active_plans_plugin.html:8]: @jiromaykin to clean up and after that this PR can be merged

Comment (jiromaykin) [reply src/open_inwoner/templates/cms/collaborate/active_plans_plugin.html:8]: @vaszig I will clean it up and request re-review.

**Pull request merged**